src/components/organism/WithdrawPoolLiquidity/index.tsx

```diff
@@ -0,0 +1,357 @@
+
+      const assetTokenInfo: any = await api.query.assets.asset(selectedTokenB.assetTokenId);
+      const assetTokenInfoMinBalance = assetTokenInfo.toHuman().minBalance.replace(/[, ]/g, "");
+
+      const lpTokenTotalAsset: any = await api.query.poolAssets.asset(location?.state?.lpTokenId);
+
+      const lpTotalAssetSupply = lpTokenTotalAsset.toHuman()?.supply.replace(/[, ]/g, "");
+
+      const lpTokenUserAccount = await api.query.poolAssets.account(
+        location?.state?.lpTokenId,
+        selectedAccount?.address
+      );
+      const lpTokenUserAsset = lpTokenUserAccount.toHuman() as LpTokenAsset;
+      const lpTokenUserAssetBalance = parseInt(lpTokenUserAsset?.balance.replace(/[, ]/g, ""));
+
+      setLpTokensAmountToBurn(lpTokenUserAssetBalance.toFixed(0));
+
+      if (res && slippageValue) {
+        const wndInPool = new Decimal(res[0].replace(/[, ]/g, ""));
+        const wndOut = wndInPool
+          .mul(new Decimal(lpTokenUserAssetBalance).toNumber())
+          .dividedBy(new Decimal(lpTotalAssetSupply).toNumber())
+          .floor()
+          .toNumber();
+
+        const assetInPool = new Decimal(res[1].replace(/[, ]/g, ""));
+        const assetOut = assetInPool
+          .mul(new Decimal(lpTokenUserAssetBalance).toNumber())
+          .dividedBy(new Decimal(lpTotalAssetSupply).toNumber())
+          .floor()
+          .toNumber();
+
+        const wndOutFormatted = formatDecimalsFromToken(wndOut, selectedTokenA?.nativeTokenDecimals);
+        const assetOutFormatted = formatDecimalsFromToken(assetOut, selectedTokenB?.decimals);
+
+        const wndOutSlippage = calculateSlippageReduce(wndOutFormatted, slippageValue);
+        const wndOutSlippageFormatted = formatInputTokenValue(wndOutSlippage, selectedTokenA?.nativeTokenDecimals);
+
+        const assetOutSlippage = calculateSlippageReduce(assetOutFormatted, slippageValue);
+        const assetOutSlippageFormatted = formatInputTokenValue(assetOutSlippage, selectedTokenB?.decimals);
+
+        setMinimumTokenAmountExceeded(assetInPool.sub(assetOut).lessThanOrEqualTo(assetTokenInfoMinBalance));
+
+        setSelectedTokenNativeValue({
+          tokenValue: formatDecimalsFromToken(wndOut, selectedTokenA?.nativeTokenDecimals),
+        });
+
+        setNativeTokenWithSlippage({ tokenValue: parseInt(wndOutSlippageFormatted) });
+
+        setSelectedTokenAssetValue({ tokenValue: formatDecimalsFromToken(assetOut, selectedTokenB?.decimals) });
+        setAssetTokenWithSlippage({ tokenValue: parseInt(assetOutSlippageFormatted) });
+      }
+    }
+  };
+
+  useEffect(() => {
+    setSelectedTokenA({
+      nativeTokenSymbol: tokenBalances?.tokenSymbol as NativeTokenProps,
+      nativeTokenDecimals: tokenBalances?.tokenDecimals as NativeTokenProps,
+    });
+  }, [tokenBalances]);
+
+  useEffect(() => {
+    handleWithdrawPoolLiquidityGasFee();
+  }, [nativeTokenValue && assetTokenValue]);
```

Comment:
Why do u have nativeTokenValue && assetTokenValue? Shouldn't be [nativeTokenValu, assetTokenValue]